InnerEye/ML/SSL/lightning_containers/ssl_image_classifier.py

```diff
@@ -64,7 +64,4 @@ def get_data_module(self) -> InnerEyeDataModuleTypes:
         return self.data_module
 
     def get_trainer_arguments(self) -> Dict[str, Any]:
-        trained_kwargs = {}
-        if self.is_debug_model:
-            trained_kwargs.update({"limit_train_batches": 1, "limit_val_batches": 1})
-        return trained_kwargs
+        return {}
```

Comment:
What is the point of this function?

Reply:
Added a comment in the code. Effectively, need to undo the get_trainer_arguments from the superclass.